app/stac_api/templates/style/hover.css

```diff
@@ -14,6 +14,7 @@
     top:4em;
     left:70px;
     z-index:1000;
+    color: black;
```

Comment:
@msom I just test it the current behavior on sys-data.dev.bgdi.ch and it seems that if your browser is set to darkmode, this will greatly help as the background of the search help text is white in this case and make the help text non readable, on non darkmode the background is grey and the text black and is perfectly readable. This change should not affect non dark mode, but maybe double check if you haven't yet